### PR TITLE
MOTOR-843 Add support for the comment field to all helpers

### PR DESCRIPTION
**Pull request by blink1073**
Some of these were added as part of #158

### CODE REVIEW

**Comment by juliusgeo** [comment]
One thing that could be added is unit tests for the docstrings, similar to my "comment" field PR: https://github.com/juliusgeo/mongo-python-driver/blob/5160ae8da3e802baae44e2a0a5bb84e346cd67be/test/test_comment.py on line 71.

**Comment by juliusgeo** [comment]
One thing that could be added is unit tests for the docstrings, similar to my "comment" field PR: https://github.com/juliusgeo/mongo-python-driver/blob/5160ae8da3e802baae44e2a0a5bb84e346cd67be/test/test_comment.py on line 71.

**Comment by blink1073** [comment]
We had to skip that test in #158 because it is checking the signature, and we would have had to update all of the signatures in Motor to be explicit instead of using args and kwargs.

File: motor/docstrings.py [hunk 1]
```diff
@@ -662,6 +696,9 @@ async def create_x_index():
   {'_id_': {'key': [('_id', 1)]},
    'x_1': {'unique': True, 'key': [('x', 1)]}}
 
+
```
Comment (juliusgeo) [inline motor/docstrings.py:699]: I think this blank line should be placed below.

Comment (blink1073) [reply motor/docstrings.py]: Done

**Review by juliusgeo — APPROVED** [review]
LGTM!

**Review by ShaneHarvey — APPROVED** [review]

**Pull request merged**